Adyen/Util/HMACValidator.cs

```diff
@@ -72,7 +72,13 @@ public string GetDataToSign(NotificationRequestItem notificationRequestItem)
             return String.Join(":", signedDataList);
         }
 
-        public bool IsValidHmac(NotificationRequestItem notificationRequestItem, string key)
+        /// <summary>
+        /// Validates a regular webhook with the given <paramref name="hmacKey"/>.
+        /// </summary>
+        /// <param name="notificationRequestItem"><see cref="NotificationRequestItem"/>.</param>
+        /// <param name="hmacKey">The HMAC key, retrieved from the Adyen Customer Area.</param>
+        /// <returns>A return value indicates the HMAC validation succeeded.</returns>
+        public bool IsValidHmac(NotificationRequestItem notificationRequestItem, string hmacKey)

```

Comment:
How would the merchant know not to use this over the BalancePlatform function? Would they know the distinction between the functions?

Reply:
I'm also thinking about more webhook collections that we still need to Add.

Reply:
Are we going to build a class+method and README for every notification collection we'll support?

Reply:
Yeah I think for now that's the best tbh. I can see management notifications getting pretty big and combining that with balanceplatform would not be ideal I think.

Reply:
Plus as it stand we have a distinct group of; payments, management and platform webhooks. I'd assume this will stay the same?